compiler/compile/OSRData.cpp

```diff
@@ -624,7 +624,10 @@ TR_OSRMethodData::TR_OSRMethodData(int32_t _inlinedSiteIndex, TR::ResolvedMethod
         _linkedToCaller(false),
         slot2ScratchBufferOffset(comp()->allocator()),
         _numSymRefs(0),
-        bcInfoHashTab(comp()->allocator()),  bcLiveRangeInfoHashTab(comp()->allocator()), argInfoHashTab(comp()->allocator()) {};
+        bcInfoHashTab(comp()->allocator()),
+        bcLiveRangeInfoHashTab(comp()->allocator()),
+        bcPendingPushLivenessInfoHashTab(comp()->allocator()),
+        argInfoHashTab(comp()->allocator()) {};
```

Comment:
if we are breaking this into two lines the indenting isn't right. I do see some strange HashIndex stuff because of the CS2 HashTable usage - would these be better as an std::map to simplify that code?

Reply:
It would likely be simpler and easy to replace. It should be noted that the existing CS2 hash table will not overwrite entries, keeping the original value instead. This behaviour should be matched, or OSR logic will have to be updated. I believe this should be done in a separate PR.